Makefile

```diff
@@ -1,6 +1,9 @@
 deps:
-	sudo apt install python3.8 python3-pip nodejs libpython3.8-dev libpq-dev graphicsmagick
-	python3.8 -m pip install poetry
+	sudo apt install python3.8 python3-pip libpython3.8-dev libpq-dev graphicsmagick
+	curl -fsSL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+	sudo apt-get install -y nodejs
+	curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3.8 -	
```

Comment:
wanna add comments linking back to the documentation where you got the node/poetry install commands from? makes it easy to go back and reference